datamade/la-metro-councilmatic
# Add Monkruman to project keyring

## Overview

See title.

### Checklist


## Testing Instructions

 * If you haven't already, [install Blackbox](https://github.com/datamade/how-to/blob/main/security/gpg/blackbox.md#install-blackbox).
 * Check out this branch and run `blackbox_postdeploy`. Confirm the command succeeds.
 * Run `blackbox_shred_all_files` clean up the unencrypted files from your machine.

Connects #814.
